recipes/pyophidia/build.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env python
```

Comment:
Can we drop `build.sh` in favor of:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L29